solidity/contracts/test/BridgeStub.sol

```diff
@@ -46,6 +46,21 @@ contract BridgeStub is Bridge {
         }
     }
 
+    function setProcessedMovedFundsSweepRequests(
+        BitcoinTx.UTXO[] calldata utxos
+    ) external {
+        for (uint256 i = 0; i < utxos.length; i++) {
+            uint256 utxoKey = uint256(
+                keccak256(
+                    abi.encodePacked(utxos[i].txHash, utxos[i].txOutputIndex)
+                )
+            );
+            self.movedFundsSweepRequests[utxoKey].state = MovingFunds
+                .MovedFundsSweepRequestState
+                .Processed;
+        }
+    }
```

Comment:
When working on https://github.com/keep-network/tbtc-v2/issues/173, let's try splitting `BridgeStub` into smaller, per-test-file stubs. The contract size is not a problem but we have so many functions in the stub now that it's sometimes hard to figure out which one is needed to test which functionality, and looking at the test code which one is from stub and which one is from the actual contract